perun-dispatcher/src/test/java/cz/metacentrum/perun/dispatcher/unit/EventServiceResolverTest.java

```diff
@@ -29,10 +33,39 @@
 	@Autowired
 	private EventServiceResolver eventServiceResolver;
 
+	@Autowired
+	PerunBl perun;
+
+	@Before
+	public void setupTests() throws Exception {
+
+		super.setupTests();
+
+		// create some group in there
+		group1 = new Group("falcon", "desc");
+		group1 = perun.getGroupsManager().createGroup(sess, vo1, group1);
+		// create user in the VO
+		// skip the xEntry (authorization check),
+		// could skip the xBl a go directly to xImpl to avoid writing audit
+		// log
+		user1 = new User(0, "firstName", "lastName", "", "", "");
+		user1 = perun.getUsersManagerBl().createUser(sess, user1);
+		// make the user the member of the group
+		member1 = perun.getMembersManagerBl().createMember(sess, vo1, user1);
+		member1.setStatus("VALID");
+		perun.getGroupsManagerBl().addMember(sess, group1, member1);
+
+		// assign the group to resource
+		perun.getResourcesManagerBl().assignGroupToResource(sess, group1, resource1);
+
+	}
+
 	@Test
 	public void parseEventTest() throws ServiceNotExistsException, InvalidEventMessageException, PrivilegeException {
 		System.out.println("EventServiceResolver.parseEventTest()");
 
+		// FIXME - přesunout setup sem, jinde member není potřeba
```

Comment:
Comments shouldn't be in Czech.

Reply:
I've removed the comment. I do not think it is bad to have set up in a separate method. It can be used in the future.